engine/src/main/java/org/terasology/config/LaunchPopupConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This seems a little too specific - only one option and you won't get many more options for the launch popup :) I'd remove this class and add the field to `TelemetryConfig`.